python/lancedb/table.py

```diff
@@ -121,6 +124,7 @@ def create_index(self, num_partitions=256, num_sub_vectors=96):
         self._dataset.create_index(
             column=VECTOR_COLUMN_NAME,
             index_type="IVF_PQ",
+            metric=metric,
```

Comment:
do we need to do checks on metric string? or normalize it to upper / lower case?

Reply:
we do this in Lance already. And the Rust code converts it to lowercase